docs/source/examples/Notebook/Distributing Jupyter Extensions as Python Packages.ipynb

```diff
@@ -0,0 +1,280 @@
+    "### Defining the server extension\n",
+    "This example shows that the server extension and its `load_jupyter_server_extension` function are defined in the `__init__.py` file.\n",
+    "#### `my_module/__init__.py`\n",
+    "\n",
+    "```python\n",
+    "def _jupyter_server_extension_paths():\n",
+    "    return [{\n",
+    "        \"module\": \"my_module\"\n",
+    "    }]\n",
+    "\n",
+    "\n",
+    "def load_jupyter_server_extension(nbapp):\n",
+    "    nbapp.log.info(\"my module enabled!\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install and enable the server extension\n",
+    "Which a user can install with:\n",
+    "```\n",
+    "jupyter serverextension enable --py my_module [--sys-prefix|--user]\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example - Server extension and nbextension"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating a Python package with a server extension and nbextension\n",
+    "Here is another server extension, with a front-end module.  It assumes this directory structure:\n",
+    "\n",
+    "```\n",
+    "- setup.py\n",
+    "- MANIFEST.in\n",
+    "- my_fancy_module/\n",
+    "  - __init__.py\n",
+    "  - static/\n",
+    "    index.js\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "### Defining the server extension and nbextension\n",
+    "This example again shows that the server extension and its `load_jupyter_server_extension` function are defined in the `__init__.py` file. This time, there is also a function `_jupyter_nbextension_path` for the nbextension.\n",
+    "\n",
+    "#### `my_fancy_module/__init__.py`\n",
+    "\n",
+    "```python\n",
+    "def _jupyter_server_extension_paths():\n",
+    "    return [{\n",
+    "        \"module\": \"my_fancy_module\"\n",
+    "    }]\n",
+    "\n",
+    "# Jupyter Extension points\n",
+    "def _jupyter_nbextension_paths():\n",
+    "    return [dict(\n",
+    "        section=\"notebook\",\n",
+    "        # the path is relative to the `my_fancy_module` directory\n",
+    "        src=\"static\",\n",
+    "        # directory in the `nbextension/` namespace\n",
+    "        dest=\"my_fancy_module\",\n",
+    "        # _also_ in the `nbextension/` namespace\n",
+    "        require=\"my_fancy_module/index\")]\n",
+    "\n",
+    "def load_jupyter_server_extension(nbapp):\n",
+    "    nbapp.log.info(\"my module enabled!\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install and enable the server extension and nbextension\n",
+    "\n",
+    "The user can install and enable the extensions with the following set of commands:\n",
+    "```\n",
+    "jupyter serverextension install --py my_fancy_module [--sys-prefix|--user]\n",
```

Comment:
This should be `jupyter serverextension enable --py my_fancy_module [--sys-prefix|--user]` I think. There is no `serverextension install` subcommand according to the CLI.